tests/dags/common/operators/test_postgres_result.py

```diff
@@ -5,7 +5,6 @@
 import datetime
 import os
 
-import pendulum
```

Comment:
Nice to keep only one time library in this file, and standardize on `datetime` in general.